### ISSUE TITLE
Incorrect paths when when using 'SSMParameterGroup'

### ISSUE THREAD

**Issue by pawelad** [alexcasalboni/ssm-cache-python#32]
Hi!

I'm playing around with your (very cool looking) library and I think I noticed a bug - when using `SSMParameterGroup` the `name` (path of the attributes) is incorrect (contains duplicates):

```python
group = SSMParameterGroup(base_path="/database/foo")
params = group.parameters("/production")

params[0].name  # this is '/database/foo/database/foo/production/password'
```

**Comment by alexcasalboni** [comment]
@pawelad thanks for reporting 💯 

I have fixed this bug in this PR: #35 

Will be releasing a new tag `2.8` soon (possibly including the other requests too)

**Comment by pawelad** [comment]
Awesome, thanks for the quick responses and fixes, and let me know if you need any help with the other couple feature requests : -)
